### PR TITLE
Support SNO collect rules only

### PR DESCRIPTION
**Pull request by subbarao-meduri**
Signed-off-by: Subbarao Meduri <smeduri@redhat.com>

### CODE REVIEW

File: operators/multiclusterobservability/controllers/placementrule/manifestwork.go [hunk 1]
```diff
@@ -585,20 +585,24 @@ func mergeMetrics(defaultAllowlist []string, customAllowlist []string) []string
...
 
-	mergedCollectRuleGroups := []operatorconfig.CollectRuleGroup{}
 	for _, collectRuleGroup := range defaultCollectRuleGroupList {
 		if !deletedCollectRuleGroups[collectRuleGroup.Name] {
 			mergedCollectRuleGroups = append(mergedCollectRuleGroups, collectRuleGroup)
 		}
 	}
 
-	config.CollectRulesEnabled = len(mergedCollectRuleGroups) == 2
+	config.CollectRulesEnabled = len(mergedCollectRuleGroups) > 0
```
Comment (marcolan018) [inline operators/multiclusterobservability/controllers/placementrule/manifestwork.go:605]: seems we need to remove this logic now. 
for the dynamic datasource, it always keep its' original value regardless of collect rule disable or not

Comment (subbarao-meduri) [reply operators/multiclusterobservability/controllers/placementrule/manifestwork.go]: Removed logic in latest commit.

**Pull request merged**